### ISSUE TITLE
Import Component from react-native instead of react. 

### ISSUE THREAD

**Issue by horatiua** [magicismight/react-native-toggle#4]
"dependencies": {
    "react": "15.1.0",
    "react-native": "0.28.0",
    "react-native-toggle": "^1.0.3"
  }

![2016-06-23_1933](https://cloud.githubusercontent.com/assets/3829689/16311478/76fb25aa-3979-11e6-8cab-559e174a2eeb.png)

**Comment by heldertb** [comment]
Yeah, the npm package hasn't been updated with the latest code on github some why. He fixed this.
Anyway, to fix it yourself, go to node_modules/react-native-toggle/Toggle.js and change this:
`import React, {
    requireNativeComponent,
    View,
    Component,
    PropTypes
} from 'react-native';`

To:

`import React, {Component, PropTypes} from 'react';
import {
    requireNativeComponent,
    View
} from 'react-native';`
